lm75bd/lm75bd.c

```diff
@@ -27,8 +30,19 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  uint8_t tmp_reg_pointer = LM75BD_TEMP_REG;
+  i2cSendTo(devAddr, &tmp_reg_pointer, 1);
+  uint8_t buffer[] = {0x00, 0x00};
```

Comment:
If you want to declare and initialize a buffer (of size 2) to all zeros, you can also do `uint8_t buffer[2] = {0};`. If you have a brace-enclosed list and there are fewer initialized values than the number of total elements in the list/structure, the uninitialized values will become zero (this is guaranteed by the C standard). This is particularly useful if you want to initialize a large buffer.